client/RoomClient.js

```diff
@@ -843,13 +843,70 @@ export default class RoomClient extends EventTarget
 						break;
 					}
 
-					case 'initState':
+					case 'initState': {
+						console.log('Room.js, initState:', notification);
+						break;
+					}
 					case 'updateState':
 					{
+						console.log('Room.js, updateState:', notification);
+						this.dispatchEvent(new MessageEvent(notification.method, {
+							data: notification.data,
+						}));
+						break;
+					}
+
+					case 'setState':
+					{
+						console.log('Room.js, setState:', notification)
+						this.dispatchEvent(new MessageEvent(notification.method, {
+							data: notification.data,
+						}));
+						break;
+					}
+
+					case 'getFile':
+					{
+						console.log('Room.js, getFile:', notification)
+						this.dispatchEvent(new MessageEvent(notification.method, {
+							data: notification.data,
+						}));
+						break;
+					}
+
+					case 'edit':
+					{
+						console.log('Room.js, edit:', notification)
+						this.dispatchEvent(new MessageEvent(notification.method, {
+							data: notification.data,
+						}));
+						break;
+					}
+
+					case 'getAllKeys':
```

Comment:
What's the difference between keys and files? Can that be written into the API names?

Reply:
👍

Reply:
we should just talk in "keys"